packages/gafl-webapp-service/src/pages/layout/layout.njk

```diff
@@ -9,6 +9,10 @@
 {% from "button/macro.njk" import govukButton %}
 {% from "footer/macro.njk" import govukFooter %}
 
+{% if (SHOW_WELSH_CONTENT) and (_uri.queryParams.lang == 'cy') %}
```

Comment:
Would be nice to see this as a flag in the injected data, which can then be tested, rather than logic being in the template

Reply:
@jaucourt Good shout, updated